css/css-contain/content-visibility/content-visibility-with-top-layer-and-auto-descendant.html

```diff
@@ -16,13 +16,13 @@
 </style>
 
 <div id="spacer" style="height: 100vh"></div>
-<div popover="manual" style="display: block; position: static;" id="popover">
```

Comment:
Up to you, but isn't the testcase more appealing with `style="display: block; position: static;"` ? 

Since you'd be testing the transition from an offscreen state to an onscreen state.

Reply:
Yes, I was missing the significance of that part before, will fix.